standard/statements.md

```diff
@@ -1309,6 +1311,33 @@ The order in which `foreach` traverses the elements of an array, is as follows:
 >  
 > *end example*
 
+An async enumerator may optionally expose a `DisposeAsync` method that may be invoked with no arguments and that returns something that can be `await`ed and whose `GetResult()` returns `void`.
```

Comment:
I suspect there's a cleaner way of expressing this via task types, e.g. "with a return type that is a non-generic task type".